zh/SUMMARY.md

```diff
@@ -3,113 +3,113 @@
 ## 新手入门
 
 - [了解引擎](README.md)
-    - [关于]()
+    - [关于我们]()
     - [引擎优势]()
-    - [How to learn](warn内容少,建议融入[获取帮助],去除本节)
+    - [如何学习]()
     - [获取帮助]()
     - [支持]()
 
 - [基本概念](README.md)
-    - [Main Components](info建议融到本章介绍,去除本节)
+    - [主要组件 Main Components]()
     - [导演]()
     - [场景]()
     - [精灵]()
     - [动作]()
     - [序列]()
-    - [Parent Child Relationship](warn和并列的节,不属于一类,建议融到入[节点]和[精灵]的内容中)
+    - [父子关系]()
     - [日志]()
 
 ## 引擎组件
 
 - [精灵](README.md)
-    - [What are Sprites](info内容少,建议融到本章介绍,去除本节)
+    - [精灵概念]()
     - [精灵创建]()
-    - [精灵控制](info调整了节的顺序)
-    - [精灵表单](warn翻译是否合适)
-    - [精灵帧缓存]()
-    - [多边形精灵]()
+    - [精灵表单 Using Spritesheets]()
+    - [精灵帧缓存 SpriteFrame Cache]()
+    - [精灵控制]()
+    - [多边形精灵 Polygon Sprite]()
 
 - [动作](README.md)
-    - [简介]()
+    - [动作概念]()
     - [基本动作](./action/zh.md)
     - [动作序列]()
-    - [序列原理](info翻译是否合适)
+    - [序列原理 Sequence Internals, what is really happening]()
 
 - [场景](README.md)
-    - [What is a Scene](warn内容少,建议融入到本章介绍,去除本节)
+    - [场景概念]()
     - [场景创建]()
     - [场景转换]()
 
 - [UI组件](README.md)
-    - [Widget,oh,my!](warn内容少,建议融到本章介绍,去除本节)
+    - [组件概念]()
     - [标签]()
     - [菜单]()
     - [按钮]()
     - [选框]()
-    - [载入条](warn翻译是否合适?)
-    - [滑动条](warn翻译是否合适?)
+    - [载入条 Loading Bar]()
+    - [滑动条 Slider]()
     - [文本框]()
 
-- [高级节点类型?](README.md)
-    - [本章标题翻译是否合适?]()
-    - [More than just a Node](warn内容少,建议融到本章介绍,去除本节)
+- [高级节点类型 Other Node Types](README.md)
+    - [不仅仅是节点 More than just a Node]()
     - [瓦片地图]()
     - [粒子系统]()
-    - [视差](warn翻译是否合适)
+    - [视差 Parallax]()
 
 ## 功能
 
 - [事件分发](README.md)
-    - [What is the EventDispatch mechanism](warn内容少,建议融到本章介绍,在中文文档中去除本节)
-    - [事件监听器]()
+    - [事件分发简介]()
+    - [事件监听器 5 types of event listeners]()
     - [触发优先级]()
     - [触碰事件]()
     - [键盘事件]()
     - [鼠标事件]()
     - [加速度传感器事件]()
-    - [自定义事件](info包含[Create-Custom-Events]和[Registering-events-with-the-dispatcher]两节的内容,即自定义事件的创建,注册和分发)
+    - [创建自定义事件]()
+    - [注册并分发事件 Registering Events with the Dispatcher]()
 
 - [3D支持](README.md)
-    - [Terminology](warn术语介绍,内容较少,建议融到本章介绍,去除本节)
+    - [术语]()
     - [3D精灵]()
     - [动画]()
     - [相机]()
-    - [方块纹理](warn翻译是否合适)
-    - [模拟天空](warn翻译是否合适)
+    - [方块纹理 Cubemap Texture]()
+    - [模拟天空 Skybox]()
     - [光照]()
-    - [地表](warn翻译是否合适)
+    - [地表 Terrain]()
     - [3D工具]()
-    - [高阶话题](warn翻译是否合适)
+    - [进一步讨论 Advanced 3D Topics]()
 
 - [使用脚本](README.md)
     - [脚本组件]()
 
 - [物理引擎](README.md)
-    - [Physics is scary, do I really need it? Please tell me no!](warn介绍性质,建议放于本章介绍,在中文文档中去除本节)
+    - [物理引擎简介]()
     - [术语与概念]()
     - [碰撞]()
-    - [Queries](error本节如何翻译?)
+    - [Queries 如何翻译]()
```

Comment:
how to translate

Reply:
it is a concept of physics engine? i have not idea with it, you can refer to physics engine docs to check its meaning.